front/src/modules/simulationResult/components/simulationResultsConsts.ts

```diff
@@ -21,6 +21,7 @@ export const CHART_AXES = {
 export type ChartAxes = ObjectFieldsTypes<typeof CHART_AXES>;
 export type XAxis = ChartAxes[0];
 export type YAxis = ChartAxes[1];
+export type Y2Axis = ChartAxes[1];
```

Comment:
This would be more precise. We could also remove space_height from YAxis
```suggestion
export type Y2Axis = typeof CHART_AXES.SPACE_HEIGHT;
```